client/src/legacy/TabSet.js

```diff
@@ -96,22 +119,7 @@ $.entwine('ss', function($){
         $(this).attr('href', document.location.href.replace(/#.*/, '') + matches[0]);
       });
     },
-
-    /**
-     * @func lazyLoadGridFields
-     * @desc Find all the lazy loadable gridfield in the panel and trigger their reload.
-     * @param {Object} panel
-     */
-    lazyLoadGridFields: function(panel) {
```

Comment:
Instead of removing this entirely (which could be called by someone's own project code for whatever reason they may have), move this up into the `$('.ss-tabset, .cms-tabset')` scope, and have `triggerLazyLoad` call this function instead of handling gridfields itself.